packages/libs/error-stack/tests/snapshots/doc/fmt__charset_ascii.snap

```diff
@@ -1,7 +1,7 @@
 <b>invalid input parameter</b>
-<b>|- </b><i>src/fmt/charset.rs:21:5</i>
-<b>|- </b>backtrace (1)
-<b>|- </b>suggestion: oh no, try again
+|- at <i>src/fmt/charset.rs:21:5</i>
```

Comment:
Do we want to remove the whitespace after `-` again as a drive-by?

Reply:
removed i n[20ef0a0](https://github.com/hashintel/hash/pull/1980/commits/20ef0a082978f68e92a4062a72598daa7dab4a77)